src/test/crimson/seastore/test_seastore.cc

```diff
@@ -518,3 +548,98 @@ TEST_F(seastore_test_t, simple_extent_test)
     test_obj.check_size(*seastore);
   });
 }
+
+TEST_F(seastore_test_t, fiemap_empty)
+{
+  run_async([this] {
+    auto &test_obj = get_object(make_oid(0));
+    test_obj.touch(*seastore);
+    test_obj.truncate(*seastore, 100000);
+
+    std::map<uint64_t, uint64_t> m, e;
+    m = test_obj.fiemap(*seastore, 0, 100000);
+    e[0] = 100000;
+    EXPECT_TRUE(m == e || m.empty());
```

Comment:
I'm a bit confused here, shouldn't m be empty always?

Reply:
I move store_test.cc fiemap test case here.  at that test, it does this check. But I think in our implementation, it should be empty.  I don't know why it check m ==e, so I will remove it.

Reply:
Yeah, we can be more precise in our requirements in these tests.  We can always loosen the requirements later if we need to.